Analysis/Physics/Chic2invi/Chic02ee/Chic02ee-00-00-01/src/Chic02ee.cc

```diff
@@ -383,17 +388,18 @@ iChargedGood.push_back((*itTrk)->trackId());
 } // end charged tracks
 
 m_nGoodChargedTrack = iChargedGood.size();
-m_nlptrk = iLepPGood.size();
-m_nlmtrk = iLepMGood.size(); 
+m_nlptrk = iPGood.size();
```

Comment:
1.  you had not push_back anything into this vector before this line,  it is empty here when you try to get the size of this vector !  
2.try to understand what is the real meaning and usage of this vector,  and read the example codes line by line, do it after understanding the logic.
3. I already told you this bug during the last face-to-face discussion, I should suggest you **again** that one should trust the pen not his/her memory.  Next time, **write down** the items during the discussion.